Library/ViewModels/PledgePaymentPlansViewModelTest.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+@testable import Library
+import ReactiveExtensions
+import ReactiveExtensions_TestHelpers
+import XCTest
+
+final class PledgePaymentPlansViewModelTests: TestCase {
```

Comment:
Scott, just to clarify, the PR already includes this unit tests for `PledgePaymentPlansViewModel`. If you feel there are specific areas that still need additional coverage, I’d be happy to take a look and add more tests as needed.